com.unity.netcode.gameobjects/Tests/Runtime/HRTClock.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace MLAPI.RuntimeTests
+{
+    public sealed class HRTClock : IDisposable
```

Comment:
Super minor, almost misplaced, comment: It is probably a good idea to submit additional classes like this as separate PR, prior to the dependent one.